cypress/integration/ui_tests/browse.js

```diff
@@ -0,0 +1,169 @@
+import LoginPage from '../../pages/loginPage';
+import HomePage from '../../pages/homePage';
+import BrowsePage from '../../pages/browsePage';
+import AccountPage from '../../pages/accountPage';
+import UiScenario from '../../components/ui_scenarios';
+import user from '../../fixtures/user.json';
+
+describe('Login Test', () => {
+    beforeEach(function () {
+        cy.fixture('user').then(function (user) {
+            this.user = user;
+        })
+        cy.viewport('iphone-6');
+        cy.visit('https://test.thetreedots.com/');
+        try {
+            cy.clearLocalStorage();
+            cy.clearCookies();
+        } catch (e) {
+            console.log(e)
+        }
+        cy.reload();
+    });
+
+    it('Check Price on Browse', function () {
+        UiScenario.login(this.user.emailMY, this.user.password);
+        cy.get(HomePage.accountMenu).should('exist');
+        cy.get('.expand_icon').click();
+        cy.get('.text-grey > :nth-child(1) > .q-item__section').click();
```

Comment:
Move this selector to Page Object Model please.

Reply:
Thank you so much. Already change it.